setup.py

```diff
@@ -47,8 +47,7 @@
         "types-requests",
     ],
     'test': [
-        'pytest',
-        'pytest-cov',
+        'pytest>=4.6',
```

Comment:
Note: pytest-4.6 has been packaged since Ubuntu 20.04. But I think it's okay because this dependency is for developers, not users.

| Ubuntu LTS | pytest version|
| --- | --- |
| 18.04 | 3.3.2 |
| 20.04 | 4.6.9 |
| 22.04 | 6.2.5 |

refs: https://packages.ubuntu.com/jammy/python3-pytest